GLOSSARY.md

```diff
@@ -0,0 +1,284 @@
+  * [Microtask Queue](#microtask-queue)
+* [Links](#links)
+
+## Error Symposium
+
+An ongoing series of facilitated discussions led by **@groundwater**, originally
+conceived at the August 2015 Node Collaborator Summit at Mozilla SF. Their goal
+is to produce recommendations on "best practices" for error handling in Node.
+
+Notably, the conversation has been informed by the [foundational work][joyent-errors]
+written by **@davepacheco**. This document was paraphrased by **@chrisdickinson** as
+the part of the Node.js [error documentation][node-errors].
+
+[The results of their latest meeting are here][symposium-findings].
+
+### Operational Error
+
+An error that represents a known state of the program. To paraphrase the
+[joyent error guide][joyent-errors]:
+
+> Operational errors represent run-time problems experienced by correctly-written
+> programs. These are not bugs in the program. In fact, these are usually
+> problems with something else: the system itself (e.g., out of memory or too
+> many open files), the system's configuration (e.g., no route to a remote host),
+> the network (e.g., socket hang-up), or a remote service (e.g., a 500 error,
+> failure to connect, or the like).
+
+Notably, operational errors that are not accounted for by the programmer are
+considered unexpected errors.
+
+### Programmer Error
+
+An error that represents a mistake on the programmer's behalf. To paraphrase the
+[joyent error guide][joyent-errors]:
+
+> Programmer errors are bugs in the program. These are things that can always
+> be avoided by changing the code. They can never be handled properly (since by
+> definition the code in question is broken).
+
+By definition, these errors are unexpected.
+
+### Expected versus Unexpected
+
+Errors may be "expected" — anticipated by the program author at time of writing —
+or "unexpected" — unforeseen by the author.
+
+Programmer errors are _always_ unexpected (or else the programmer would not
+have made them!) Operational errors _may be_ unexpected — there may be an
+unaccounted-for operational state that the program enters. Unexpected errors
+are dangerous, and may cause the program to perform undesirable operations. The
+error symposium recommends that programs encountering unexpected errors should
+crash immediately.
+
+∅               | Programmer Error   | Operational Error
+--------------- | ------------------ | ------------------
+**Expected**    | :no_entry_sign:    | :white_check_mark:
+**Unexpected**  | :white_check_mark: | :white_check_mark:
+
+## Post Mortem Analysis
+
+The practice of debugging a program after it has already crashed by inspecting
+the serialized state of the program at the time of error. This usually leverages
+tools like `lldb`, `gdb`, or `mdb` to inspect a "core" (defined below.)
+
+### Abort
+
+The [`abort()`][abort-syscall] system call. Accessible to Node via
+`process.abort()`. Called by V8 when `--abort-on-uncaught-exception` is enabled
```

Comment:
I mean the fact that the default behavior in Node for `uncaughtException`s is to print the stack trace and exit the process.So it's `--abort-on-uncaught-exception` changes how it behaves with regard to post-mortem analysis but not the actual behavior of the code.